addons/l10n_ro/partner_view.xml

```diff
@@ -1,15 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <openerp>
-	<data>
-		<record model="ir.ui.view" id="res_partner_form_ro">
-			<field name="name">res.partner.form.ro</field>
-			<field name="inherit_id" ref="base.view_partner_form"/>
```

Comment:
Are you just modifying the indentation? This should have been changed at 436fc5b so I think you need to rebase your branch.